Dockerfile

```diff
@@ -18,52 +18,19 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
 # Set Timezone
 RUN echo "date.timezone = Europe/London" > /usr/local/etc/php/conf.d/timezone_set.ini
 
-###########################################################################################
-# Run test suite
-###########################################################################################
-
-FROM base AS test
-
-# Remove the memory limit for the CLI only.
-RUN echo 'memory_limit = -1' > /usr/local/etc/php/php-cli.ini
-
-# Remove the vanilla Drupal ready to install a dev version
-RUN rm -rf ..?* .[!.]* *
-
-# Install Drupal 8.x Dev
-RUN composer create-project drupal-composer/drupal-project:8.x-dev . --stability dev --no-interaction
-
-COPY phpunit.xml web/core/phpunit.xml
-COPY docroot/modules/custom web/docroot/modules/custom
-
-RUN vendor/bin/phpunit -c web/core --testsuite unit --debug --verbose
-
 ###########################################################################################
 # Create runtime image
 ###########################################################################################
 
```

Comment:
Is this comment still valid? It looks like it's tied to `FROM base as build` which we're removing.

Reply:
I've updated the comment to say "Copy repository files".  Ideally I want to keep all the COPY commands in this section, so they're all in one place.